pkg/dispatchcli/cmd/create.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/vmware/dispatch/pkg/api/v1"
 	"github.com/vmware/dispatch/pkg/dispatchcli/i18n"
 	"github.com/vmware/dispatch/pkg/utils"
+	"os"
```

Comment:
Std Library imports go to the first block

Some pointers : https://github.com/golang/go/wiki/CodeReviewComments#imports